Delivery/powercat-download.md

```diff
@@ -51,6 +51,7 @@ This query can be used to detect the following attack techniques and tactics ([s
 * [Exchange PowerShell snap-in being loaded](../Exfiltration/exchange-powershell-snapin-loaded.md)
 * [Exchange vulnerability creating web shells via UMWorkerProcess](../Execution/umworkerprocess-creating-webshell.md)
 * [Exchange vulnerability launching subprocesses through UMWorkerProcess](../Execution/umworkerprocess-unusual-subprocess-activity.md)
+* [Base64-encoded Nishang commands for loading reverse shell](./reverse-shell-nishang-base64.md)
```

Comment:
Similar issue with the path. It should be: ../Execution/reverse-shell-nishang-base64.md